### DISCUSSION TITLE
Reverse Proxy e Auth

### DISCUSSION THREAD

**Discussion by mikycol** [indomus/forum#450] [GENERALE - Discussioni generiche]
Ciao a tutti,
ho configurato un mio nginx per fare reverse proxy su Hassio, ora vorrei fare in maniera tale da passare il clientIp originale in maniera da bypassare l'autenticazione quando mi collego da un dispositivo in casa e dovermi autenticare quando sono fuori casa. Qualcuno ci è riuscito, ho visto nella configurazione si può attivare use_x_forwarded_for, ma se non ho capito male va in abbinamento all'utilizzo di trusted_proxies.

**Comment by snakuzzo** [comment]
Sì, devi abilitare l'XFF e configurare ad hoc HA...

https://www.home-assistant.io/integrations/http/#use_x_forwarded_for
